app/services/rss_reader.rb

```diff
@@ -95,34 +87,16 @@ def make_from_rss_item(item, user, feed)
     article
   end
 
-  def get_host_without_www(url)
-    url = "http://#{url}" if URI.parse(url).scheme.nil?
-    host = URI.parse(url).host.downcase
-    host.start_with?("www.") ? host[4..] : host
-  end
-
-  def medium_reply?(item)
-    get_host_without_www(item.url.strip) == "medium.com" &&
-      !item[:categories] &&
-      content_is_not_the_title?(item)
-  end
-
-  def content_is_not_the_title?(item)
-    # [[:space:]] removes all whitespace, including unicode ones.
-    content = item.content.gsub(/[[:space:]]/, " ")
-    title = item.title.delete("…")
-    content.include?(title)
-  end
-
-  def article_exists?(user, item)
-    title = item.title.strip.gsub('"', '\"')
-    feed_source_url = item.url.strip.split("?source=")[0]
-    relation = user.articles
-    relation.where(title: title).or(relation.where(feed_source_url: feed_source_url)).exists?
-  end
-
   def report_error(error, metadata)
     Honeybadger.context(metadata)
     Honeybadger.notify(error)
   end
+
+  def get_item_count_error(feed)
+    if feed
+      feed.entries ? feed.entries.length : "no count"
```

Comment:
I'm wondering about the conditional here. I thought `entries` always returns an array, will it really return `nil` if there are items?

Reply:
🤷 it might not be true anymore, hard to verify with a regression test. I don't want to poke the bear :D